Makefile

```
@@ -2,9 +2,11 @@ clone-snapshots:
 	@echo "	⏬  Cloning snapshots..."
 	@command git clone git@github.com:salemove/ios-widgets-snapshots.git SnapshotTests/__Snapshots__
 
+source_branch := $(shell git rev-parse --abbrev-ref HEAD)
 clone-snapshots-ci:
 	@echo "	⏬  Cloning snapshots..."
 	@command git clone $(repo_url) SnapshotTests/__Snapshots__
+	@command cd SnapshotTests/__Snapshots__ && ./checkout.sh $(SOURCE_BRANCH) $(destination_branch)
 
 commit-snapshots:
 	@echo "	💾  Commiting snapshots..."
```

Not relevant, but still a chance to correct the typo:
```suggestion
	@echo "	💾  Committing snapshots..."
```

---

Fixed 11155152c4c53f2e65cdabe6eee49d91da78ed0d